src/ONYXKEYS.js

```diff
@@ -90,6 +90,9 @@ export default {
     // Indicates which locale should be used
     PREFERRED_LOCALE: 'preferredLocale',
 
+    /** List of transactionIDs in process of rejection */
```

Comment:
Sorry to be crazy picky, but can you use the `//` style comments to be consistent with the other comments in this file?